projects/plugins/jetpack/modules/custom-css/csstidy/class.csstidy_print.php

```diff
@@ -160,228 +173,237 @@ function formatted_page($doctype='xhtml1.1', $externalcss=true, $title='', $lang
+					$out .= $template[4] . $this->htmlsp( $token[1], $plain ) . ':' . $template[5];
 					break;
 
 				case VALUE:
-					$out .= $this->_htmlsp($token[1], $plain);
-					if ($this->_seeknocomment($key, 1) == SEL_END && $this->parser->get_cfg('remove_last_;')) {
-						$out .= str_replace(';', '', $template[6]);
+					$out .= $this->htmlsp( $token[1], $plain );
+					if ( $this->seeknocomment( $key, 1 ) == SEL_END && $this->parser->get_cfg( 'remove_last_;' ) ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
+						$out .= str_replace( ';', '', $template[6] );
 					} else {
 						$out .= $template[6];
 					}
 					break;
 
 				case SEL_END:
 					$out .= $template[7];
-					if ($this->_seeknocomment($key, 1) != AT_END)
+					if ( $this->seeknocomment( $key, 1 ) != AT_END ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseNotEqual
 						$out .= $template[8];
+					}
 					break;
 
 				case AT_END:
-					$out = & $output;
-					$out .= $template[10] . str_replace("\n", "\n" . $template[10], $in_at_out);
+					$out       = & $output;
+					$out      .= $template[10] . str_replace( "\n", "\n" . $template[10], $in_at_out );
 					$in_at_out = '';
-					$out .= $template[9];
+					$out      .= $template[9];
 					break;
 
 				case COMMENT:
-					$out .= $template[11] . '/*' . $this->_htmlsp($token[1], $plain) . '*/' . $template[12];
+					$out .= $template[11] . '/*' . $this->htmlsp( $token[1], $plain ) . '*/' . $template[12];
 					break;
 			}
 		}
 
-		$output = trim($output);
+		$output = trim( $output );
 
-		if (!$plain) {
+		if ( ! $plain ) {
 			$this->output_css = $output;
-			$this->_print(true);
+			$this->_print( true );
 		} else {
 			// If using spaces in the template, don't want these to appear in the plain output
-			$this->output_css_plain = str_replace('&#160;', '', $output);
+			$this->output_css_plain = str_replace( '&#160;', '', $output );
 		}
 	}
 
 	/**
 	 * Gets the next token type which is $move away from $key, excluding comments
-	 * @param integer $key current position
-	 * @param integer $move move this far
+	 *
+	 * @param integer $key current position.
+	 * @param integer $move move this far.
 	 * @return mixed a token type
 	 * @access private
 	 * @version 1.0
 	 */
-	function _seeknocomment($key, $move) {
-		$go = ($move > 0) ? 1 : -1;
-		for ($i = $key + 1; abs($key - $i) - 1 < abs($move); $i += $go) {
-			if (!isset($this->tokens[$i])) {
+	public function seeknocomment( $key, $move ) {
+		$go = ( $move > 0 ) ? 1 : -1;
+		for ( $i = $key + 1; abs( $key - $i ) - 1 < abs( $move ); $i += $go ) { // phpcs:ignore Generic.CodeAnalysis.ForLoopWithTestFunctionCall.NotAllowed
+			if ( ! isset( $this->tokens[ $i ] ) ) {
 				return;
 			}
-			if ($this->tokens[$i][0] == COMMENT) {
-				$move += 1;
+			if ( $this->tokens[ $i ][0] == COMMENT ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
```

Comment:
As with line 256, if the array did happen to contain `"7"` I think it'd be a bug to consider that as being `COMMENT`.